datadog_lambda/tracing.py

```diff
@@ -89,7 +90,39 @@ def _context_obj_to_headers(obj):
     }
 
 
-def extract_dd_trace_context(event):
+def extract_context_from_http_event(event):
+    headers = event.get("headers", {})
+    lowercase_headers = {k.lower(): v for k, v in headers.items()}
+
+    trace_id = lowercase_headers.get(TraceHeader.TRACE_ID)
+    parent_id = lowercase_headers.get(TraceHeader.PARENT_ID)
+    sampling_priority = lowercase_headers.get(TraceHeader.SAMPLING_PRIORITY)
+
+    return trace_id, parent_id, sampling_priority
+
+
+def extract_context_from_sqs_event(event):
+    first_record = event["Records"][0]
+    msg_attributes = first_record.get("messageAttributes", {})
+    dd_json_data = msg_attributes.get("_datadog", {}).get("StringValue", r"{}")
+    dd_data = json.loads(dd_json_data)
+    trace_id = dd_data.get(TraceHeader.TRACE_ID)
+    parent_id = dd_data.get(TraceHeader.PARENT_ID)
+    sampling_priority = dd_data.get(TraceHeader.SAMPLING_PRIORITY)
+
+    return trace_id, parent_id, sampling_priority
+
+
+def extract_context_from_lambda_context(lambda_context):
```

Comment:
Do you mind adding a docstring or comment explaining what use case this is for? It seems to be much less explanatory compared to the http and sqs ones.